tests/filecheck/transforms/convert_onnx_to_linalg.mlir

```diff
@@ -74,6 +74,13 @@
 // CHECK-NEXT:  %res_max_pool_single_out_3 = linalg.fill ins(%res_max_pool_single_out_2 : f64) outs(%res_max_pool_single_out_1 : tensor<1x16x4x4xf32>) -> tensor<1x16x4x4xf32>
 // CHECK-NEXT:  %res_max_pool_single_out_4 = linalg.pooling_nchw_max {"dilations" = dense<1> : tensor<2xi64>, "strides" = dense<3> : tensor<2xi64>} ins(%t26, %res_max_pool_single_out : tensor<1x16x14x14xf32>, tensor<3x3xf32>) outs(%res_max_pool_single_out_3 : tensor<1x16x4x4xf32>) -> tensor<1x16x4x4xf32>
 
+%t20, %t21, %t22 = "test.op"() : () -> (tensor<1x1x5x5xf32>, tensor<1x1x3x3xf32>, none)
```

Comment:
why none?

Reply:
Ah, I see! It seems like we should check in the lowering whether the operand is `none` type or not, and not add the linalg.add if it's none. So ideally two checks one with none and one with a real shape

Reply:
erm i used a test cases for that, but we should probably check none and add if not now i think

Reply:
coool

Reply:
Code looks great, could you please add a test case for it? Then we should be GTG

Reply:
done :)